modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverAliasPropertiesTest.java

```diff
@@ -0,0 +1,97 @@
+package org.testcontainers.jdbc;
+
+import static java.util.Arrays.asList;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+
+import java.sql.SQLException;
+import java.util.EnumSet;
+import java.util.Properties;
+
+import org.apache.commons.dbutils.QueryRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+@RunWith(Parameterized.class)
+public class JDBCDriverAliasPropertiesTest {
```

Comment:
This class should be a unit test. We should test that aliases map to the right db type/image, there is no need to actually start the DB